src/reconcile/rec_track.c

```diff
@@ -15,11 +15,11 @@
 #define WT_OVFL_SIZE(p, s) (sizeof(s) + 2 * sizeof(void *) + (p)->addr_size + (p)->value_size)
 
 /*
- * __wt_ovfl_track_init --
+ * __ovfl_track_init --
  *     Initialize the overflow tracking structure.
  */
-int
-__wt_ovfl_track_init(WT_SESSION_IMPL *session, WT_PAGE *page)
+static int
+__ovfl_track_init(WT_SESSION_IMPL *session, WT_PAGE *page)
```

Comment:
Should we add an "\_\_ut_" version of this function to the end of the file, similarly to the one for discard, so that we can keep the unit tests that you were planning to remove?

Reply:
Done